obspy/io/reftek/packet.py

```diff
@@ -154,7 +154,7 @@ def __str__(self, compact=False):
             sta = (self.station_name.strip() +
                    self.station_name_extension.strip())
             info = ("{:04d} {:2s} {:4s} {:2d} {:4d} {:4d} {:2d} {:2s} "
-                    "{:5s} {:4d}         {!s}").format(
+                    "{:5s} {:4.1f}        {!s}").format(
```

Comment:
`200.0` is already `5.1f` actually, this spec will not ensure a guaranteed maximum fixed width, unfortunately. This is only for the info-printing part, though, I think? We could make more room for the sampling rate, I guess, as long as we don't exceed a  total line length of lets say 70 or so max